common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -0,0 +1,113 @@
+"""Simple utility functions that operate on course metadata.
```

Comment:
According to https://github.com/edx/edx-platform/wiki/Python-Guidelines, the one-line summary should be on the same line as the """